### ISSUE TITLE
Error in command `howdoi print in python`

### ISSUE THREAD

**Issue by CaedenPH** [gleitz/howdoi#470]
#### What happened:
```py
howdoi print in python
'd' Signed integer decimal.  
'i' Signed integer decimal.  
'o' Signed octal value. (1)
'u' Obsolete type – it is identical to 'd'. (7)
'x' Signed hexadecimal (lowercase). (2)
'X' Signed hexadecimal (uppercase). (2)
'e' Floating point exponential format (lowercase).  (3)
'E' Floating point exponential format (uppercase).  (3)
'f' Floating point decimal format.  (3)
'F' Floating point decimal format.  (3)
'g' Floating point format. Uses lowercase exponential format if exponent is less than -4 or not less than precision, decimal format otherwise.  (4)
'G' Floating point format. Uses uppercase exponential format if exponent is less than -4 or not less than precision, decimal format otherwise.  (4)
'c' Single character (accepts integer or single character string).   
'r' String (converts any Python object using repr()).   (5)
's' String (converts any Python object using str()).    (6)
```
#### What you expected to happen:
Something like "`print()`"

#### Output with `--explain`
The same error

**Comment by gleitz** [comment]
That is unfortunate. It seems that the [search results in Google](https://www.google.com/search?q=site:stackoverflow.com%20print%20in%20python&hl=en) lists https://stackoverflow.com/questions/20450531/python-operator-in-print-statement as the top result.

I get a little bit better results with `howdoi print string in python 3`, but ultimately it is up to the search engine which answers ranks first.

**Comment by CaedenPH** [comment]
> That is unfortunate. It seems that the [search results in Google](https://www.google.com/search?q=site:stackoverflow.com%20print%20in%20python&hl=en) lists https://stackoverflow.com/questions/20450531/python-operator-in-print-statement as the top result.
> 
> 
> 
> I get a little bit better results with `howdoi print string in python 3`, but ultimately it is up to the search engine which answers ranks first.

Oh, this isn't some sort of weird/broken logging you guys have implemented?

**Comment by gleitz** [comment]
Yes the results are chosen by the search engine (configurable with `--engine`) and then by the top rated answer on Stack Overflow.
